src/scss/includes/panels/favorite_panel.scss

```diff
@@ -165,7 +165,7 @@
 
 .favorite_panel__item__more_container:hover:not(.favorite_item--removed) {
   background: $surface;
```

Comment:
perhaps adding "cursor: pointer;" to this container

Reply:
@patfrat, it's already in is children .favorite_panel__item__info 
did you mean inside the autocomplete ?